### PR TITLE
[logs] Fix a few log messages related to duration measurement

### PR DESCRIPTION
**Pull request by romanz**
- Remove fixed-width formatting for the duration value (`%15d` -> `%d`), for better consistency with other [similar](https://github.com/bitcoin/bitcoin/search?q=dms&unscoped_q=dms) logging statements.
- Correct the logging statement that starts with a space (`" block index %15dms\n"`) to start with a verb. 
- Add `" in "` between the operation description and the millisecond duration.

### CODE REVIEW

**Comment by romanz** [comment]
Note: Travis and AppVeyor fail due to seemingly unrelated issues:
https://ci.appveyor.com/project/DrahtBot/bitcoin/build/master.1848?fullLog=true
https://travis-ci.org/bitcoin/bitcoin/jobs/435323308

**Comment by romanz** [comment]
Updated the PR's description to contain more information about the issues it fixes.

**Comment by romanz** [comment]
Also, I'm not sure how to test it (other than manually running `bitcoind` and inspecting the logs)...

**Comment by promag** [comment]
How about this one:
https://github.com/bitcoin/bitcoin/blob/1e8f88e071019907785b260477bd359bef6f9a8f/src/validation.cpp#L2312

This only benefits the human eye as the durations can be extracted either way. Consistency is a plus though.

Retracted from NACK to -0.

**Comment by romanz** [comment]
Good suggestion!
Will update the PR in the following days.

**Comment by romanz** [comment]
I have updated (most) the benchmarking-related code in this PR to be consistent with the format used at [src/validation.cpp](https://github.com/bitcoin/bitcoin/blob/1e8f88e071019907785b260477bd359bef6f9a8f/src/validation.cpp#L2312), except the code at [src/wallet/wallet.cpp](https://github.com/bitcoin/bitcoin/pull/14358/files#diff-b2bb174788c7409b671c46ccc86034bd) (since it uses [CWallet::WalletLogPrintf()](https://github.com/bitcoin/bitcoin/blob/7990bb5b61cb343e47a0333dfc0dd9f0cd4ae5a8/src/wallet/wallet.h#L1123)).

**Comment by romanz** [comment]
@promag Ping :)

File: src/init.cpp [hunk 1]
```diff
@@ -1508,7 +1508,7 @@ bool AppInitMain()
             }
 
             fLoaded = true;
-            LogPrintf(" block index %15dms\n", GetTimeMillis() - load_block_index_start_time);
+            LogPrint(BCLog::BENCH, "- Loaded block index: %dms\n", GetTimeMillis() - load_block_index_start_time);
```
Comment (promag) [inline src/init.cpp]: @romanz I didn't meant to change the log category. I think some of these messages must/should be present in the log regardless of the enabled logging categories.

Comment (romanz) [reply src/init.cpp]: Updated the PR to use the original logging categories.

**Comment by romanz** [comment]
Updated the PR to use the original logging categories.

**Comment by romanz** [comment]
Ping (rebased on latest master edc715240).

**Comment by romanz** [comment]
@promag Please let me know if there is a problem with this PR.

**Pull request closed without merging**